test/integration/api/datasets.js

```diff
@@ -0,0 +1,311 @@
+const appRoot = require('app-root-path');
+const { testService } = require('../setup');
+const testData = require('../../data/xml');
+const config = require('config');
+const { Form } = require('../../../lib/model/frames');
+const { getOrNotFound } = require('../../../lib/util/promise');
+const { omit } = require('ramda');
+// eslint-disable-next-line import/no-dynamic-require
+const { createEntityFromSubmission } = require(appRoot + '/lib/worker/entity');
+
+// TODO merge with test/integration/api/forms/dataset.js
+
+describe('projects/:id/datasets', () => {
```

Comment:
Add a top-level describe block to this file to make it easy to run dataset-related tests.